lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -164,6 +163,13 @@ def transform_service_pay(service_pay_source)
 
       private
 
+      def populate_te(lh_request_body)
```

Comment:
as noted in comments above, this is a temporary workaround for a current Lighthouse bug that was blocking testing